CONCURRENCY.md

```diff
@@ -207,4 +219,4 @@ that are potentially parts of shared cyclic data should be zeroed out once no lo
  If atomic reference value is attempted to be set to non-frozen value runtime exception is thrown.
```

Comment:
If you attempt to set an atomic reference value to non-frozen, a value runtime exception will be thrown.

Reply:
I haven't updated content in this article. Will request proofread separately. 
I just fixed formatting there.